css/style.css

```diff
@@ -24,6 +24,27 @@ a.btn:hover
     background-color: lightgray;
 }
 
+/* replaces <big> tag */
+em.big
+{
+    font-style: normal;
+    font-size: larger;
+}
+
+/* replaces <tt> tag */
```

Comment:
I was a bit worried about interference with other styling. And it would be odd if the TT macro would be a `<code>` tag, while the CODE macros is some `<code class="">` tag.